both/collections/competition.js

```diff
@@ -100,9 +120,7 @@ Competitions.attachSchema({
         return "missingRegistrationOpenDate";
       }
 
-      if(registrationOpenDate.getTime() < registrationCloseDate.getTime()) {
-        return null;
-      } else {
+      if(registrationOpenDate.getTime() >= registrationCloseDate.getTime()) {
```

Comment:
Seems better to me than the weird `.field` stuff! Meteor's apis can be pretty confusing/schizophrenic feeling to me at times. Maybe I'm just not always "getting" it.